### PR TITLE
Edited username checking in gold auth command

### PR DESCRIPTION
**Pull request by SilverTactic**
Using <code>Users.getExact()</code> is better, since it returns undefined if the exact user isn't present on the server. (ex. <code>Users.getExact('siiilver')</code> would return undefined if I changed names or went offline)

### CODE REVIEW

**Comment by SilverTactic** [comment]
Okay, I take that back. <code>Users()</code> and <code>Users.get()</code> are the same, but using <code>Users.getExact()</code> is better, since it returns undefined if the exact user isn't present on the server. (ex. <code>Users.getExact('siiilver')</code> would return undefined if I changed names or went offline)

File: chat-plugins/gold.js [hunk 1]
```diff
@@ -82,8 +82,8 @@ exports.commands = {
 				var rank = row[i].split(',')[1].replace("\r", '');
 				var person = row[i].split(',')[0];
 				function nameColor (name) {
-					if (Users(name) && Users(name).connected) {
-						return '<b><i><font color="' + Gold.hashColor(toId(name)) + '">' + Tools.escapeHTML(name) + '</font></i></b>';
+					if (Users.getExact(name)) {
+						return '<b><i><font color="' + Gold.hashColor(toId(name)) + '">' + Tools.escapeHTML(Users.getExact(name).name) + '</font></i></b>';
```
Comment (panpawn) [inline chat-plugins/gold.js]: This line can stay as it was

File: chat-plugins/gold.js [hunk 2]
```diff
@@ -1414,4 +1414,4 @@ function htmlfix(target) {
             target = target.substring(0, target.indexOf(fixings[u])) + '< ' + target.substring(target.indexOf(fixings[u]) + 1);
     }
     return target;
-}
\ No newline at end of file
+}
```
Comment (panpawn) [inline chat-plugins/gold.js]: This trailing whitespace was intentional for ``npm test`` to have one less thing to complain about

Comment (SilverTactic) [inline chat-plugins/gold.js]: Oh

Comment (panpawn) [inline chat-plugins/gold.js]: I'll fix this later

**Pull request merged**